nest/proto.c

```diff
@@ -89,12 +97,58 @@ proto_log_state_change(struct proto *p)
 	{
 	  p->last_state_name_announced = name;
 	  PD(p, "State changed to %s", proto_state_name(p));
+	  notify_bgp_stat_daemon(p);
 	}
     }
   else
     p->last_state_name_announced = NULL;
 }
 
+/**
+ * @proto: pointer to struct proto
+ * Function sends a UDP message to the BGP stat daemon notifying the state of the router
+ * This function will be called whenever the router state changes
+ */
+static void
+notify_bgp_stat_daemon(struct proto *p){
```

Comment:
please follow the spacing and hanging brackets style presented in the rest of the file